cli/src/main.rs

```diff
@@ -1,10 +1,10 @@
-use db::PgPool;
 use clap::{Parser, Subcommand};
+use db::PgPool;
 use std::fs;
+use std::io::Read;
 use std::io::Write;
-use uuid::Uuid;
 use std::path::Path;
-use std::io::Read;
+use uuid::Uuid;
```

Comment:
Oh! My editor runs `cargo fmt` by default and this re-arranged some of these, and removed some commas down below and all that!

We'll probably wire up fmt to CI to make sure its run, but we don't need to worry about that now!